src/libraries/restore/runtime/runtime.depproj

```diff
@@ -80,6 +86,9 @@
         <DestinationSubDirectory>include/%(RecursiveDir)</DestinationSubDirectory>
       </NativeBinPlaceItem>
     </ItemGroup>
+    <ItemGroup Condition="'$(RuntimeFlavor)' == 'Mono' and '$(TargetsMobile)' != 'true'">
```

Comment:
You don't need the `RuntimeFlavor` condition, the target already contains that condition.

Reply:
Thanks, done.